docs/reference/client.md

```diff
@@ -4,159 +4,141 @@ A list of the functions available with `import truss` and their arguments and pr
 
 ### Truss creation
 
-    cleanup()
-        Cleans up .truss directory.
+#### cleanup()
 
-    from_directory(truss_directory: str) -> truss.truss_handle.TrussHandle
-        Get a handle to a Truss. A Truss is a build context designed to be built
-        as a container locally or uploaded into a baseten serving environment.
+Cleans up .truss directory.
+    
+#### from_directory(truss_directory: str) -> truss.truss_handle.TrussHandle
 
-        Args:
-            truss_directory (str): The local directory of an existing Truss
-        Returns:
-            TrussHandle
+Get a handle to a Truss. A Truss is a build context designed to be built as a container locally or uploaded into a baseten serving environment.
```

Comment:
perhaps you meand `model serving environment` instead of `baseten serving environment`.

Reply:
If this is automatically derived from the code then we may need to update it there as well. Those comments may be out of date.